examples/src/main/java/com/basistech/rosette/examples/NameDeduplicationExample.java

```diff
@@ -38,11 +39,11 @@ public static void main(String[] args) {
     }
 
     private void run() throws IOException {
-        String nameDedupeData = "John Smith,Johnathon Smith,Fred Jones";
+        List<String> nameDedupeData = Arrays.asList("John Smith", "Johnathon Smith", "Fred Jones");
```

Comment:
This will break the python code that populates the data string.  It looks for the variable name and replaces anything after the = with the new string.